configs/milvus.yaml

```diff
@@ -68,7 +68,9 @@ minio:
   port: 9000 # Port of MinIO/S3
   accessKeyID: minioadmin # accessKeyID of MinIO/S3
   secretAccessKey: minioadmin # MinIO/S3 encryption string
-  useSSL: false # Access to MinIO/S3 with SSL
+  ssl:
+    enabled: false # Access to MinIO/S3 with SSL
+    caCert: /path/to/ca.pem # path to your CACert file
```

Comment:
We try using unified configurations for all the third-parties ssl:
For Etcd: https://github.com/milvus-io/milvus/blob/8f7019468f9fb2ed82f6dabeabbf85701bc34f08/configs/milvus.yaml#L36

For Kafka: https://github.com/milvus-io/milvus/blob/8f7019468f9fb2ed82f6dabeabbf85701bc34f08/configs/milvus.yaml#L133

The path of CACert file is named "tlsCaCert", please follow this rule.